zboxcore/sdk/allocation_test.go

```diff
@@ -0,0 +1,120 @@
+package sdk
+
+import (
+	"github.com/0chain/gosdk/core/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const tokenUnit = 10000000000.0
+
+func Test_getMinWriteRead_success(t *testing.T) {
+	var (
+		ssc = newTestAllocation()
+		err error
+	)
+
+	ssc.initialized = true
+	sdkInitialized = true
+	assert.NotNil(t, ssc.BlobberDetails)
+
+	minW, minR, err := ssc.GetMinWriteRead()
+	require.NoError(t, err)
+	assert.Equal(t, 800000/tokenUnit, minW)
```

Comment:
Why are you using both  `require` and `assert`? Can't you just stick to `require` and use EqualValue, InDelta or InEplsion etc... 
I think we want the test to stop as soon as there is an error.

Reply:
Resolved